roles/core/time/tasks/main.yml

```diff
@@ -21,14 +21,15 @@
 - name: Set timezone
   timezone:
     name: "{{ time_zone }}"
+  when: time_zone is defined
```

Comment:
If we define time_zone in defaults/main.yml, then we would not need to add this test.